src/otserv.cpp

```diff
@@ -37,96 +33,67 @@
 #include "script.h"
 #include <fstream>
 
-DatabaseTasks g_databaseTasks;
-Dispatcher g_dispatcher;
-Scheduler g_scheduler;
+extern Scripts* g_scripts; // Holds '*.lua' scripts from '/data'. Extern means it's defined elsewhere.
+
+DatabaseTasks g_databaseTasks;	// Runs on separate thread, handles database queries
+Scheduler g_scheduler;			// Runs on separate thread, handles planned executions
+Dispatcher g_dispatcher;		// Runs on separate thread, handles immediate executions
```

Comment:
The formatting here is suboptimal

Reply:
Hey! Thanks.

Care to explain how?
I think we should always see externs before everything else.
These are just declarations, also, so it doesn't really affect performance.

Also, I just didn't want to stick yet another line into the source for these short comments.

Reply:
GitHub uses tab-size 8 instead of 4, that's the root of the problem.
https://github.com/otland/forgottenserver/blob/96153c4565d0ecf6f9537eb7cda5fa886f26e94f/src/otserv.cpp?ts=4
This is how the code _**actually**_ looks like.